tensorflow_datasets/image/patchcamelyon.py

```diff
@@ -0,0 +1,154 @@
+_PCAM_TEST_LABELS_FILENAME = "camelyonpatch_level_2_split_test_y.h5.gz"
+_PCAM_IMAGE_SIZE = 96
+_PCAM_IMAGE_SHAPE = (_PCAM_IMAGE_SIZE, _PCAM_IMAGE_SIZE, 3)
+_TRAIN_EXAMPLES = 262144
+_VALID_EXAMPLES = 32768
+_TEST_EXAMPLES = 32768
+
+_PCAM_CITATION = """\
+@ARTICLE{Veeling2018-qh,
+  title         = "Rotation Equivariant {CNNs} for Digital Pathology",
+  author        = "Veeling, Bastiaan S and Linmans, Jasper and Winkens, Jim and
+                   Cohen, Taco and Welling, Max",
+  month         =  jun,
+  year          =  2018,
+  archivePrefix = "arXiv",
+  primaryClass  = "cs.CV",
+  eprint        = "1806.03962"
+}
+"""
+
+_CAM_CITATION = """\
+@article{Litjens2018,
+    author = {Litjens, G. and Bándi, P. and Ehteshami Bejnordi, B. and Geessink, O. and Balkenhol, M. and Bult, P. and Halilovic, A. and Hermsen, M. and van de Loo, R. and Vogels, R. and Manson, Q.F. and Stathonikos, N. and Baidoshvili, A. and van Diest, P. and Wauters, C. and van Dijk, M. and van der Laak, J.},
+    title = {1399 H&E-stained sentinel lymph node sections of breast cancer patients: the CAMELYON dataset},
+    journal = {GigaScience},
+    volume = {7},
+    number = {6},
+    year = {2018},
+    month = {05},
+    issn = {2047-217X},
+    doi = {10.1093/gigascience/giy065},
+    url = {https://dx.doi.org/10.1093/gigascience/giy065},
+    eprint = {http://oup.prod.sis.lan/gigascience/article-pdf/7/6/giy065/25045131/giy065.pdf},
+}
+"""
+
+
+class PatchCAMELYON(tfds.core.GeneratorBasedBuilder):
+    """PatchCAMELYON"""
+    URL = _PCAM_URL
+    ZENODO_URL = "https://zenodo.org/record/2546921/files/"
+
+    VERSION = tfds.core.Version("1.0.0")
+
+    def _info(self):
+        return tfds.core.DatasetInfo(
+            builder=self,
+            description=("The PatchCAMELYON dataset for identification of breast cancer metastases in lymph nodes. "
+                         "This dataset has been extracted from the larger CAMELYON dataset of 1399 whole-slide images, "
+                         "which created for the CAMELYON challenges at ISBI 2016 and 2017."
+                         "It contains 96x96 RGB patches of normal lymph node and tumor tissue in a roughly 50/50 "
+                         "distributions. It packs the clinically-relevant task of metastasis detection into a "
+                         "straight-forward image classification task, akin to CIFAR-10 and MNIST. This increases the "
+                         "ease of use by removing the complexity of handling large whole-slide images."),
+            features=tfds.features.FeaturesDict({
+                "image": tfds.features.Image(shape=_PCAM_IMAGE_SHAPE),
+                "label": tfds.features.ClassLabel(num_classes=2),
+            }),
+            supervised_keys=("image", "label"),
+            urls=[_PCAM_URL, _CAM_URL],
+            citation=_PCAM_CITATION + _CAM_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
```

Comment:
Please check your code style with pylint for tf code style.
Check this [link](https://www.tensorflow.org/community/contribute/code_style)